AndroidTest/app/src/main/java/cloudant/com/androidtest/keyprovider/KeyDataTests.java

```diff
@@ -0,0 +1,135 @@
+package cloudant.com.androidtest.keyprovider;
+
+import android.test.AndroidTestCase;
+
+import com.cloudant.sync.datastore.encryption.KeyData;
+
+public class KeyDataTests extends AndroidTestCase {
```

Comment:
Changes requested in comment included in https://github.com/cloudant/sync-android/commit/dd276d263bfc9c5ddebb047cecd27230cb074f63